.github/workflows/v3_audit-prod.yml

```
@@ -0,0 +1,47 @@
+  REGISTRY: ghcr.io
+  IMAGE_NAME: ${{ github.repository }}/audit
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    permissions:
+      contents: read
+      packages: write
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v3
```

actionlint (1.7.7)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)